docs/pbc/all/ratings-reviews/202212.0/third-party-integrations/configure-bazaarvoice.md

```diff
@@ -44,15 +44,28 @@ The following video explains how you can create an API key if it is not availabl
   </video>
 </figure>
 
-8. In **Settings > Select Services**, select the services that you need:
+8. In **sFTP Settings** fill out BazaarVoice's SFTP credentials if you want to track whether the products that have more UGC have a better conversion rate than products without many ratings and reviews.
```

Comment:
@jeremyfourna isn't there **Settings > Select Services** anymore? Where can I check the new Configuration page for Bazaarvoice?